src/components/EarnDashboard/components/ProviderDetails/OpportunityRow.tsx

```diff
@@ -140,7 +140,7 @@ export const OpportunityRow: React.FC<
             <NestedAsset
               key={rewardAssetId}
               isClaimableRewards={isClaimableRewards}
-              isExternal={opportunity.isReadOnly}
+              isExternal={opportunity.isReadOnly && !asset?.isPool}
```

Comment:
Fix to not show the read-only icon for read-only assets that route to swapper for Portals swapping